web/docs/Custom-Extensions.mdx

```diff
@@ -50,19 +50,27 @@ Add the following dependencies:
 <GradleCode>
 
 ```kotlin title="build.gradle.kts"
-compileOnly("com.google.auto.service:auto-service-annotations:1.0")
+compileOnly("com.google.auto.service:auto-service-annotations:1.1.1")
 //either for java sources:
-annotationProcessor("com.google.auto.service:auto-service:1.0")
+annotationProcessor("com.google.auto.service:auto-service:1.1.1")
 //or for kotlin sources (requires kapt gradle plugin):
-kapt("com.google.auto.service:auto-service:1.0")
+kapt("com.google.auto.service:auto-service:1.1.1")
+//or for ksp(requires ksp gradle plugin).
+//Due to the missing official KSP implementation of AutoService a custom plugin is required:
```

Comment:
I don't think this sentence is necessary for the docs - googles implementation for kapt isn't really official either

Reply:
Agree. Removed. 
Anyway, this section will be updated from time to time. And one day we will remove `ksp("dev.zacsweers.autoservice:auto-service-ksp:version")`